Library/Formula/libstfl.rb

```diff
@@ -36,165 +39,27 @@ def install
-+#
-+#ifeq ($(FOUND_SWIG)$(FOUND_PERL5),11)
-+#include perl5/Makefile.snippet
-+#endif
- 
- ifeq ($(FOUND_SWIG)$(FOUND_PYTHON),11)
- include python/Makefile.snippet
-diff --git a/perl5/Makefile.PL b/perl5/Makefile.PL
-index 1c4fa58..501ff7d 100644
---- a/perl5/Makefile.PL
-+++ b/perl5/Makefile.PL
-@@ -3,6 +3,6 @@ WriteMakefile(
-         "NAME"      => "stfl",
- 	# The -D_LARGEFILE64_SOURCE -D_GNU_SOURCE are needed by perl (not STFL!) on some systems
- 	"CCFLAGS"   => "-pthread -I.. -D_LARGEFILE64_SOURCE -D_GNU_SOURCE",
--        "LIBS"      => ["-lpthread -lncursesw"],
-+        "LIBS"      => ["-lpthread -lncurses"],
-         "OBJECT"    => "stfl_wrap.o ../libstfl.a"
- );
-diff --git a/python/Makefile.snippet b/python/Makefile.snippet
-index 8fd4052..0eb3f16 100644
---- a/python/Makefile.snippet
-+++ b/python/Makefile.snippet
-@@ -27,8 +27,12 @@ install: install_python
- 
- python/_stfl.so python/stfl.py python/stfl.pyc: libstfl.a stfl.h python/stfl.i swig/*.i
- 	cd python && swig -python -threads stfl.i
--	gcc -shared -pthread -fPIC python/stfl_wrap.c -I/usr/include/python$(PYTHON_VERSION) \
--		-I. libstfl.a -lncursesw -o python/_stfl.so
-+	$(CC) -shared -pthread -fPIC python/stfl_wrap.c \
-+		-L$(shell python-config --prefix)/lib \
-+		$(shell python-config --include) \
-+		$(shell python-config --ldflags) \
-+		-liconv \
-+		-I. libstfl.a -lncurses -o python/_stfl.so
- 	cd python && python -c 'import stfl'
- 
- install_python: python/_stfl.so python/stfl.py python/stfl.pyc
-diff --git a/ruby/Makefile.snippet b/ruby/Makefile.snippet
-index e0cf641..c837563 100644
---- a/ruby/Makefile.snippet
-+++ b/ruby/Makefile.snippet
-@@ -25,7 +25,7 @@ install: install_ruby
- 
- ruby/build_ok: libstfl.a stfl.h ruby/stfl.i swig/*.i
- 	cd ruby && swig -ruby stfl.i && ruby extconf.rb
--	$(MAKE) -C ruby clean && $(MAKE) -C ruby LIBS+="../libstfl.a -lncursesw" CFLAGS+="-pthread -I.." DLDFLAGS+="-pthread" DESTDIR=$(DESTDIR) prefix=$(prefix) sitedir=$(prefix)/$(libdir)/ruby
-+	$(MAKE) -C ruby clean && $(MAKE) -C ruby LIBS+="../libstfl.a -lncurses -lruby -liconv" CFLAGS+="-pthread -I.." DLDFLAGS+="-pthread" DESTDIR=$(DESTDIR) prefix=$(prefix) sitedir=$(prefix)/$(libdir)/ruby
- 	touch ruby/build_ok
- 
- install_ruby: ruby/build_ok
-diff --git a/stfl.pc.in b/stfl.pc.in
-index 0e4cd84..4d9cca8 100644
---- a/stfl.pc.in
-+++ b/stfl.pc.in
-@@ -9,5 +9,5 @@ Name: STFL
- Description: Structured Terminal Forms Language/Library
- Version: @VERSION@
- Libs: -L${libdir} -lstfl
--Libs.private: -lncursesw
-+Libs.private: -lncurses
- Cflags: -I${includedir}
-diff --git a/stfl_internals.h b/stfl_internals.h
-index 3f9f45b..1559626 100644
---- a/stfl_internals.h
-+++ b/stfl_internals.h
-@@ -28,7 +28,7 @@ extern "C" {
- #endif
- 
- #include "stfl.h"
--#include <ncursesw/ncurses.h>
-+#include <ncurses.h>
- #include <pthread.h>
- 
- struct stfl_widget_type;
--- 
-1.7.11.1
+    inreplace "perl5/Makefile", "Network/Library", etc
```

Comment:
Original patch looks like it killed perl off because of this. I assume etc isn't the right replacement if anything actually is.